pcsx2-qt/Debugger/MemorySearchWidget.cpp

```diff
@@ -453,7 +453,7 @@ std::vector<SearchResult> startWorker(DebugInterface* cpu, const SearchType type
 			isSigned ? searchWorker<s32>(cpu, searchResults, type, comparison, start, end, value.toInt(nullptr, base)) : searchWorker<u32>(cpu, searchResults, type, comparison, start, end, value.toUInt(nullptr, base));
 			break;
 		case SearchType::Int64Type:
-			isSigned ? searchWorker<s64>(cpu, searchResults, type, comparison, start, end, value.toLong(nullptr, base)) : searchWorker<s64>(cpu, searchResults, type, comparison, start, end, value.toULongLong(nullptr, base));
```

Comment:
I believe this code is still broken on Windows when searching for a signed Int64. `value.toLong(..)` returns `long`, 4 bytes on Windows.
Would you be able to change `value.toLong(nullptr, base)` to `value.toLongLong(nullptr, base)` ?